src/ssSeasonsUtil.lua

```diff
@@ -47,7 +47,8 @@ ssSeasonsUtil.seasons = {
 }
 
 function ssSeasonsUtil:load(savegame, key)
-    self.daysInSeason = ssStorage.getXMLFloat(savegame, key .. ".settings.daysInSeason", 10)
+    self.daysInSeason = math.floor(ssStorage.getXMLFloat(savegame, key .. ".settings.daysInSeason", 12) / 3) * 3
+	self.daysInMonth = self.daysInSeason / 3;
```

Comment:
Not sure we agree about adding months. Everything we did is designed around seasons